### ISSUE TITLE
`dot` breaks Graphviz

### ISSUE THREAD

**Issue by jayvdb** [ysdexlic/dot#1]
The binary `dot` conflicts with Graphviz `dot`, which is fairly widely used.
Perhaps this tool's bin could be renamed to `dotm` ?

**Comment by ysdexlic** [comment]
Fair point, I'll have to change a few other bits in my dotfiles to match so I'll get on it this weekend!
